web/src/components/__tests-data__/columns.tsx

```diff
@@ -1,14 +1,88 @@
-import { Column, ColumnsId } from '../netflow-table-header';
+import { Column, ColumnsId, FilterType } from '../../utils/columns';
 
 export const ColumnsSample: Column[] = [
-  { id: ColumnsId.date, name: 'Date & time', isSelected: true, defaultOrder: 1 },
-  { id: ColumnsId.srcpod, name: 'Src pod', isSelected: true, defaultOrder: 2 },
-  { id: ColumnsId.dstpod, name: 'Dst pod', isSelected: true, defaultOrder: 3 },
-  { id: ColumnsId.srcnamespace, name: 'Src namespace', isSelected: true, defaultOrder: 4 },
-  { id: ColumnsId.dstnamespace, name: 'Dst namespace', isSelected: true, defaultOrder: 5 },
-  { id: ColumnsId.srcport, name: 'Src port', isSelected: true, defaultOrder: 6 },
-  { id: ColumnsId.dstport, name: 'Dst port', isSelected: true, defaultOrder: 7 },
-  { id: ColumnsId.protocol, name: 'Protocol', isSelected: true, defaultOrder: 8 },
-  { id: ColumnsId.bytes, name: 'Bytes', isSelected: true, defaultOrder: 9 },
-  { id: ColumnsId.packets, name: 'Packets', isSelected: true, defaultOrder: 10 }
+  {
+    id: ColumnsId.timestamp,
+    name: 'Date & time',
+    isSelected: true,
+    defaultOrder: 1,
+    filterType: FilterType.DATETIME
+  },
+  {
+    id: ColumnsId.srcpod,
+    name: 'Src pod',
+    isSelected: true,
+    defaultOrder: 2,
+    filterType: FilterType.POD
+  },
+  {
+    id: ColumnsId.dstpod,
+    name: 'Dst pod',
+    isSelected: true,
+    defaultOrder: 3,
+    filterType: FilterType.POD
+  },
+  {
+    id: ColumnsId.srcnamespace,
+    name: 'Src namespace',
+    isSelected: true,
+    defaultOrder: 4,
+    filterType: FilterType.NAMESPACE
+  },
+  {
+    id: ColumnsId.dstnamespace,
+    name: 'Dst namespace',
+    isSelected: true,
+    defaultOrder: 5,
+    filterType: FilterType.NAMESPACE
+  },
+  {
+    id: ColumnsId.srcaddr,
+    name: 'Src address',
+    isSelected: true,
+    defaultOrder: 6,
+    filterType: FilterType.ADDRESS
+  },
+  {
+    id: ColumnsId.dstaddr,
+    name: 'Dst address',
+    isSelected: true,
+    defaultOrder: 7,
+    filterType: FilterType.ADDRESS
+  },
+  {
+    id: ColumnsId.srcport,
+    name: 'Src port',
+    isSelected: true,
+    defaultOrder: 8,
+    filterType: FilterType.NUMBER
+  },
+  {
+    id: ColumnsId.dstport,
+    name: 'Dst port',
+    isSelected: true,
+    defaultOrder: 9,
+    filterType: FilterType.NUMBER
```

Comment:
srcport and dstport are no longer numbers since it can display the service name.

Reply:
Should we move to an autocomplete like protocols here ?

With the ability to pick from a list or just type a number

Reply:
I prefer autocomplete with typing, but I'm okay either way.